### PR TITLE
Added Jels88

### PR DESCRIPTION
**Pull request by Jels-kb**
## Description

Added support for the Jels88 PCB.
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/jels/jels88/config.h [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#define MATRIX_COL_PINS \
+    { C7, C6, F7, F6, F5, F4, B1, D2, D3 }
+
+/* ws2812b RGB LEDs */
+#define RGB_DI_PIN B6
+
+#ifdef RGBLIGHT_ENABLE
+#    define RGBLED_NUM 20  // Number of LEDs
+#    define RGBLIGHT_SLEEP
+#    define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/jels/jels88/config.h:44]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/jels/jels88/info.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
+{
+    "keyboard_name": "Jels88",
+    "url": "",
+    "maintainer": "Jels",
+    "width": 18.25,
+    "height": 6.25,
```
Comment (drashna) [inline keyboards/jels/jels88/info.json:6]:
```suggestion
```

File: keyboards/jels/jels88/rules.mk [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (drashna) [inline keyboards/jels/jels88/rules.mk:11]:
```suggestion
```

File: keyboards/jels/jels88/rules.mk [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```
Comment (drashna) [inline keyboards/jels/jels88/rules.mk:17]: There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

File: keyboards/jels/jels88/rules.mk [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no 	    # Enable keyboard backlight functionality on B7 by default
+UNICODE_ENABLE = no         # Unicode
```
Comment (drashna) [inline keyboards/jels/jels88/rules.mk:27]:
```suggestion
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no 	    # Enable keyboard backlight functionality
```

File: keyboards/jels/jels88/rules.mk [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+AUDIO_ENABLE = no           # Audio output on port C6
```
Comment (drashna) [inline keyboards/jels/jels88/rules.mk:28]:
```suggestion
AUDIO_ENABLE = no           # Audio output
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/jels/jels88/jels88.c [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    if (res) {
+        writePin(CAPS_LED, led_state.caps_lock);
+        writePin(SCROLL_LED, led_state.scroll_lock);
+    }
+    return res;
+}
```
Comment (fauxpark) [inline keyboards/jels/jels88/jels88.c:38]: Can be replaced with: https://docs.qmk.fm/#/feature_led_indicators

File: keyboards/jels/jels88/rules.mk [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no 	    # Enable keyboard backlight functionality on B7 by default
+UNICODE_ENABLE = no         # Unicode
+AUDIO_ENABLE = no           # Audio output
+RGBLIGHT_ENABLE = yes       # Enable WS2812 RGB underlight.
```
Comment (fauxpark) [inline keyboards/jels/jels88/rules.mk:20]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/jels/jels88/jels88.c (hunk 7 above)
Comment (Jels-kb) [reply keyboards/jels/jels88/jels88.c:38]: I prefer using this method, is nice if people want to change the functionality of these pins later on.

File: keyboards/jels/jels88/rules.mk [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no 	    # Enable keyboard backlight functionality on B7 by default
+UNICODE_ENABLE = no         # Unicode
+AUDIO_ENABLE = no           # Audio output
+RGBLIGHT_ENABLE = yes       # Enable WS2812 RGB underlight.
```
Comment (fauxpark) [inline keyboards/jels/jels88/rules.mk:22]:
```suggestion
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output

```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/jels/jels88/jels88.c (hunk 7 above)
Comment (fauxpark) [reply keyboards/jels/jels88/jels88.c:38]: It's still possible to do that.

Comment (Jels-kb) [reply keyboards/jels/jels88/jels88.c:38]: More what I mean by that is that using this method, It makes it a little bit simpler changing the functionality of these pins.

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**